perun-wui-registrar/src/main/resources/cz/metacentrum/perun/wui/registrar/client/resources/PerunRegistrarTranslation_cs.properties

```diff
@@ -129,7 +129,9 @@ cantExtendMembershipInsufficientLoa=Pro prodloužení členství nemáte dostate
 cantBecomeMember=Nemůžete se registrovat do {0}
 cantBecomeMemberLoa=<br/>Poskytovatel Vaší identity, kterého jste použili při přihlášení ({0}) neposkytuje informaci o stupni Vašeho ověření (LoA = Level of Assurance). Prosím požádejte svého poskytovatele identity o zveřejěnní této informace nebo použijte jiného poskytovatele identity.
 cantBecomeMemberInsufficientLoa=<br/>Pro zobrazení registračního formuláře nemáte dostatečně ověřenou identitu (LoA = Level of Assurance). Kontaktujte svého poskytovatele identity ({0}) a prokažte mu svoji identitu předložením oficiálního dokladu (občanský průkaz, pas). Pokud máte k dispozici jinou identitu s vyšším stupněm ověření, zkuste se přihlásit pomocí ní.
-missingRequiredData=<h4>Nemůžete se registrovat</h4><p><br/>Poskytovatel Vaší identity, kterého jste použili při přihlášení ({0}) neposkytuje ověřená data vyžadovaná registračním formulářem. Prosím požádejte svého poskytovatele identity, aby zveřejnil následující atributy nebo použijte jiného poskytovatele identity.
+noPrefilledUneditableRequiredData=<h4>Nemůžete se registrovat</h4><p><br/>Následující povinné formulářové položky nemohou být automaticky předvyplněny a jejich hodnota je needitovatelná. Prosím kontaktujte administrátora.
```

Comment:
Maybe less technical version could be: `Následující povinné položky formuláře nemohly být automaticky předvyplněny a jejich hodnotu nelze volně zadat. Prosím kontaktujte administrátora.`.

I didn't see it in action, but is there an option how to contact the administrator? Like its part of exception dialog or do they have to write email to contacts displayed on the bottom of the page?

Reply:
A less technical message was applied. 

I think there is no option how to directly contact the administrator through this exception panel. Do you think we should somehow add this option to the registration form?

Reply:
I tested it and there is "report error" button displayed below these messages, so it should be ok.